Resources/doc/reference/installation.rst

```diff
@@ -42,9 +42,8 @@ Edit the ``config.yml`` file and add these lines:
         strict_variables: "%kernel.debug%"
 
         #sonata
-        form:
-            resources:
-                - 'SonataFormatterBundle:Form:formatter.html.twig'
```

Comment:
I think you shouldn't remove this, just comment it out and add a comment explaining when to use it.